package.json

```diff
@@ -9,6 +9,8 @@
     "axios": "^0.21.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-dropdown": "^1.9.0",
```

Comment:
what do all these dependencies do?  I know we will need some dependencies (like probably un uploader) but what's the "react-dropdown" and "@babel/preset-react"  I think babel is already added as part of the create-react-app config, so do we need an extra dependency here?

Reply:
its a part of the Dropdown component and it was required.
When I ran it it gave me an error and my research gave me this as a fix